backend/src/resolvers/household.ts

```diff
@@ -1,27 +1,25 @@
-import Pool from '../../config/dbConnect';
-import { GetHousehold, CreateHousehold, EditHousehold, DeleteHousehold } from '../handlers/householdHandler';
+import * as householdHandler from '../handlers/householdHandler';
 import { GetUsers } from '../handlers/userHandler';
 
 export const resolvers = {
     Query: {
-        household: async (_: any, { houseid }: any, context) => {
-            return await GetHousehold(houseid);
+        household: async (_: any, { houseid }: {houseid: String}, context) => {
+            return await householdHandler.GetHousehold(houseid);
         }
     },
     Mutation: {
-        createHousehold: async (_: any, { household }: any): Promise<Household> => {
-            return await CreateHousehold(household);
+        createHousehold: async (_: any, { household }: {household: Household}): Promise<Household> => {
+            return await householdHandler.CreateHousehold(household);
         },
-        editHousehold: async (_: any, { household }: any): Promise<void> => {
-            return await EditHousehold(household);
+        editHousehold: async (_: any, { household }: {household: Household}): Promise<void> => { // should this be a houseid instead of a whole household?
```

Comment:
You mean the input? I think it should be a whole household because we need optional var inputs?